.github/workflows/new-bugs-announce-notifier.yml

```
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 10
     steps:
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
           node-version: 14
```

Should we also bump Node version?
According to https://nodejs.org/en/about/previous-releases Node 16 is no longer maintained, so we could either bump it to Node 18 or Node 20:
```suggestion
          node-version: 18
```
```suggestion
          node-version: 20
```
We can also handle this in a separate PR.

---

Yep, let's try 20 in this PR.